apps/settings/locales/settings.en-US.properties

```diff
@@ -1130,8 +1130,9 @@ draw-tile-borders=Draw tile borders
 draw-layer-borders=Draw layer borders
 log-animations=Log slow animations
 remote-debugging-usb=Debugging via USB
-dev-tools-wifi=DevTools via Wi-Fi
-dev-tools-qps=Pseudo-localization
+devtools-wifi=DevTools via Wi-Fi
+devtools-device-name=Device Name
```

Comment:
Nit: We only capitalize the first letter and acronyms, so `Name` should be `name`.